### ISSUE TITLE
Cannot read property 'startsWith' of undefined in quasar upgrade -i

### ISSUE THREAD

**Issue by GordonBlahut** [quasarframework/quasar#5917]
**Describe the bug**
After upgrading to @qusasar/cli@1.0.4, I ran `quasar upgrade -i`:

```
PS D:\WebClient\ClientApp> quasar upgrade -i

  Quasar CLI · Gathering information with yarn...

 @quasar/app: 1.4.1 → 1.4.2
 @quasar/quasar-app-extension-qoverlay: 1.0.0 → 1.0.1
C:\Users\Gordon\AppData\Roaming\npm\node_modules\@quasar\cli\bin\quasar-upgrade:173
      const notPinned = pkg.dependencies[dep.packageName].startsWith('^')
                                                          ^

TypeError: Cannot read property 'startsWith' of undefined
    at deps.(anonymous function).forEach.dep (C:\Users\Gordon\AppData\Roaming\npm\node_modules\@quasar\cli\bin\quasar-upgrade:173:59)
    at Array.forEach (<anonymous>)
    at upgradeQuasar (C:\Users\Gordon\AppData\Roaming\npm\node_modules\@quasar\cli\bin\quasar-upgrade:168:16)
    at Object.<anonymous> (C:\Users\Gordon\AppData\Roaming\npm\node_modules\@quasar\cli\bin\quasar-upgrade:185:1)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
```

**Codepen/jsFiddle/Codesandbox (required)**
N/A

**To Reproduce**
1. See above

**Expected behavior**
Quasar upgrade deps.

**Screenshots**

**Platform (please complete the following information):**
OS: Windows_NT(10.0.18362) - win32/x64
Node: 10.15.3
NPM: 6.13.4
Yarn: 1.21.1

**Additional context**
I am using npm for global packages and yarn for local packages.

**Comment by pdanpdan** [comment]
Please report to https://github.com/quasarframework/quasar-ui-qoverlay

**Comment by rstoenescu** [comment]
Issue is valid. Fixing

**Comment by rstoenescu** [comment]
Fixed in "@quasar/cli" v1.0.5.
Thank you for reporting!

**Comment by GordonBlahut** [comment]
Thanks for looking into it so quickly. 

I just upgraded to 1.0.5 and tried `quasar upgrade -i` again and it failed with a different error. It appears to be because the previous `quasar upgrade -i` with 1.0.4 left node_modules in a bad state (@quasar/app disappeared).

After deleting node_modules and reinstalling all local packages, 1.0.5 works as expected.
